test/functional/rpc_psbt.py

```
@@ -8,13 +8,16 @@
 from decimal import Decimal
 from itertools import product
 
+from test_framework.descriptors import descsum_create
+from test_framework.key import ECKey
 from test_framework.test_framework import BitcoinTestFramework
 from test_framework.util import (
     assert_approx,
     assert_equal,
     assert_raises_rpc_error,
     find_output
```

Ruff (0.12.2)

Trailing comma missing

Add trailing comma

(COM812)